CHANGELOG.rst

```diff
@@ -12,6 +12,13 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+*
+
+
+[1.0.0] - 2020-06-12
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```

Comment:
Please, follow the guidelines of https://keepachangelog.com/en/1.0.0/

i.e. Add a section called **Added** with the things that were added and ad a section **Removed** for the thing that was removed